source/fundamentals/connection/tls.txt

```diff
@@ -77,6 +77,26 @@ in the following ways:
    see the :manual:`SRV Connection Format </reference/connection-string/#srv-connection-format>`
    section in the Server manual.
 
+.. note:: Workaround for an "unsafe legacy renegotiation disabled" error
+
+   The {+driver-short+} depends on OpenSSL by default. You may encounter an "unsafe
+   legacy renegotiation disabled" error in certain environments when using OpenSSL 
+   versions 3.0 and later, due to outdated SSL proxies. If you encounter this error, 
+   you can set the ``SSL_OP_LEGACY_SERVER_CONNECT`` option to resolve, as shown in 
```

Comment:
S: after your first draft is done, see if any sentences can be shortened / more direct without sacrificing clarity or tone. here, you could do something like this:

```suggestion
   versions 3.0 and later, due to outdated SSL proxies. You can resolve this error by setting
   the ``SSL_OP_LEGACY_SERVER_CONNECT`` option, as shown in 
```